src/components/services/ServiceKonnector.jsx

```diff
@@ -0,0 +1,60 @@
+import styles from '../../styles/konnectorItem'
+
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { translate } from 'cozy-ui/react/I18n'
+
+import { getConnectionStatus } from '../../reducers'
+
+import { getKonnectorIcon } from '../../lib/icons'
+import { CONNECTION_STATUS } from '../../lib/CollectStore'
+
+const ServiceKonnector = ({ t, status, konnector, showKonnector }) => (
+  <li
+    onKeyDown={e => (e.keyCode === 13 ? showKonnector(konnector) : null)}
```

Comment:
What the `keyCode` `13` means ? May we use a constant ?

Reply:
It's the Enter key, to improve accessiblity. I think we need a little helper to improve this comportement, with Escape or Ctrl touch too